javascripts/endGame.js

```diff
@@ -0,0 +1,59 @@
+function checkWin() {
+  var winningTiles = $("[data-val=" + winningTileValue + "]");
+  if (winningTiles.length > 0) {
+    priorWin = true;
+    winnerTile(winningTiles);
+    endGame('win');
+  } else { return false;}
```

Comment:
Good catch! Looks like in our refactoring, we left things behind that we don't need!